xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -3024,6 +3028,12 @@ void CVideoPlayer::HandleMessages()
         SetEnableStream(m_CurrentSubtitle, false);
 
       SetSubtitleVisibleInternal(isVisible);
+
+      const auto& ss = m_SelectionStreams.Get(STREAM_SUBTITLE, GetSubtitle());
+      const auto& as = m_SelectionStreams.Get(STREAM_AUDIO, GetAudioStream());
+
+      if (!isVisible && !g_LangCodeExpander.CompareISO639Codes(ss.language, as.language))
```

Comment:
here should be a comment similar to the one added above
because browsing through the code it is not clear why there is this "oddity" condition here

Reply:
done